lib/AST/Decl.cpp

```diff
@@ -2750,6 +2750,12 @@ ValueDecl::getFormalAccessScope(const DeclContext *useDC,
                                        treatUsableFromInlineAsPublic);
 }
 
+/// Check whether it's important to respect access control restrictions
+/// in current context.
+static bool isAccessControlDisabled(const ASTContext &ctx) {
+  return !ctx.LangOpts.EnableAccessControl;
```

Comment:
Maybe this should be a method on ASTContext instead? Can you replace any existing callers with this method?